lib/api/request/requestResponse.ts

```diff
@@ -256,6 +258,12 @@ export class RequestResponse {
     this[_request].setResult(r);
   }
 
+  /**
+   * Node identifier
+   */
+  get node (): string {
```

Comment:
Can't this be as string ?

Reply:
Same answer, the cast is for the type of `global`, not for `kuzzle.id`